.travis.yml

```diff
@@ -0,0 +1,88 @@
+conditions: v1
+sudo: required
+dist: xenial
+language: node_js
+node_js:
+  - "node"
+
+addons:
+  chrome: stable
+
+services:
+  - xvfb
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+before_install:
+  - export DISPLAY=:99.0
+  - chmod +x ./scripts/travis.sh
+
+script: ./scripts/travis.sh
+
+stages:
+  - Tests
+  - Release canary
+  - Release
+
+jobs:
+  fast_finish: true
+  include:
+    # https://docs.travis-ci.com/user/build-stages/deploy-github-releases/
+    - stage: Release
+      name: "Releasing a new version"
+      if: tag IS present
+      env: TRAVIS_MODE=release
+      deploy:
+        - provider: releases
+          api_key: $GH_TOKEN
+          file_glob: true
+          file: dist/*
+          prerelease: true
+          skip_cleanup: true
+          on:
+            branch: master
+            tags: true
+        - provider: npm
+          edge: true
+          api_key: $NPM_TOKEN
+          email: $NPM_EMAIL
+          skip_cleanup: true
+          on:
+            tags: true
+            branch: master
+      after_deploy:
+        - chmod +x ./scripts/after_deploy.sh
+        - ./scripts/after_deploy.sh "ima" "$TRAVIS_TAG" "$JENKINS_TAG_TOKEN"
```

Comment:
release notes dependency and tag fix should be entered here -> see [ima example](https://github.com/kaltura/playkit-js-ima/pull/161)

Reply:
and change "ima" to your plugin name without playkit-js